doc/classes/@GlobalScope.xml

```diff
@@ -2792,10 +2792,10 @@
 			The property is shown in the editor Inspector (default).
 		</constant>
 		<constant name="PROPERTY_USAGE_CHECKABLE" value="8" enum="PropertyUsageFlags">
-			The property can be checked in the editor inspector.
+			The property can be checked in the editor Inspector.
 		</constant>
 		<constant name="PROPERTY_USAGE_CHECKED" value="16" enum="PropertyUsageFlags">
-			The property is checked in the editor inspector.
+			The property is checked in the editor Inspector.
```

Comment:
Makes the name of this class all the more questionable when it can be mistaken for the Editor dock on a glance.

Reply:
I think it's worth evaluating a bit more closely, because for most common users it would be expected for those enum constants to *mainly* refer to the Inspector dock.

Reply:
Yeah. Even the documentation of `EditorInspector` was claiming it's _the_ Inspector dock for quite a long time :joy:  It hasn't been long since the correction.

Probably clearer to use the class name directly here.

```suggestion
			The property is checked in the [EditorInspector].
```

Reply:
> Probably clearer to use the class name directly here.

It seems like it's not the only place where it'd be clearer to use that class name around the place.